MicrosoftTeams.schema.json

```diff
@@ -1189,6 +1189,9 @@
           }
         }
       }
+    },
+    "extensions": {
```

Comment:
As far as I can see, this all looks fine.  I did some basic validation on some converted manifests that we have and I see some errors related to "overriddenByRibbonApi" property but that is all so I think it is better that we get this big change in and then later we can make small tweaks if needed.  Thanks Charles!